internal/protoveneer/cmd/protoveneer/protoveneer.go

```diff
@@ -970,3 +982,12 @@ func sliceAnyError[T any](s []T, f func(T) (bool, error)) (bool, error) {
 	}
 	return false, nil
 }
+
+// inStdLib reports whether the given import path could be part of the Go standard library,
+// by reporting whether the first component lacks a '.'.
+func inStdLib(path string) bool {
+	if i := strings.IndexByte(path, '/'); i != -1 {
```

Comment:
Micro-nit: it's possible this could be a bit cleaner/shorter with `strings.Cut` now

Reply:
Will do in a followup.

Reply:
Actually, did it here.